oci/spec_opts.go

```diff
@@ -865,6 +865,59 @@ func WithAdditionalGIDs(userstr string) SpecOpts {
 	}
 }
 
+// WithAddedAdditionalGIDs adds additional groups within the container
+// based on the image's /etc/group contents.
+// The passed in groups can be either a gid or a groupname.
+func WithAddedAdditionalGIDs(groups ...string) SpecOpts {
```

Comment:
the func name and comment is a little bit confusing admitting the oci func https://github.com/containerd/containerd/blob/v1.6.6/oci/spec_opts.go#L737

Can you mention in the comment that we append `process user's groups` within the container and not just appending `/etc/group`  plz :)

Reply:
maybe rename to `WithAppendAdditionalGIDs`?

Reply:
Actually naming is a reference `WithAddedCapabilities`

Reply:
or `WithAddedAdditionalUserGIDs `

Reply:
Let's keep naming and args consistent `WithAppendAdditionalGroups(groups string...)`